### DISCUSSION TITLE
是否同时支持两套环境和只允许sql在最后一行插入该怎么配置

### DISCUSSION THREAD

**Discussion by Ants5** [hhyo/Archery#1706] [Q&A]
1、是否支持生产环境和测试环境分开成两套审核规则，如果暂时还不能实现的话，是否可以再安装部署一个goInception工具，然后通过修改什么地方的代码，能实现if判断web页面所提交的工单信息，来判别使用哪个goInception工具来执行审核等操作

2、哪里可以设置，或者goInception工具的配置文件中是否可以设置，只允许提交的sql往数据表中最后一列插入列，其他插入指定列则报错，不给予审核通过

**Comment by hhyo** [comment]
- 可以实现，goinception有会话级别的参数配置能力，感兴趣可自行研究
- 有相关参数配置，建议阅读goinception文档
